krop/src/main/java/com/avito/android/krop/util/Transformation.kt

```diff
@@ -0,0 +1,40 @@
+package com.avito.android.krop.util
+
+import android.graphics.PointF
+import android.os.Parcel
+import android.os.Parcelable
+
+@Suppress("unused")
+data class Transformation(
```

Comment:
Transformation is immutable now.
It stands for temporary saved state, eligible for restoring soon.